src/playground/standalone-player.jsx

```diff
@@ -0,0 +1,105 @@
+import VirtualMachine from 'scratch-vm';
+import RenderWebGL from 'scratch-render';
+import AudioEngine from 'scratch-audio';
+import {SVGRenderer, BitmapAdapter} from 'scratch-svg-renderer';
+import storage from '../lib/storage';
+
+// This file is an example of how to create a standalone, full screen
+// minimal scratch player without the editor view.
+
+const EXAMPLE_PROJECT = 10128067; // 'Make it Dance' Project
+
+window.onload = function () {
+
+    // Instantiate the VM.
+    const vm = new VirtualMachine();
+    vm.attachV2BitmapAdapter(new BitmapAdapter());
+    vm.attachV2SVGAdapter(new SVGRenderer());
+
+    // Initialize storage
+    storage.addOfficialScratchWebStores();
+    vm.attachStorage(storage);
+
+    // Compatibility mode will set the frame rate to 30 TPS,
+    // which is the standard for the scratch player.
+    vm.setCompatibilityMode(true);
```

Comment:
Why isn't this the default? Took me and @z a minute to figure out why everything was sped up (before adding this).

Reply:
Made an issue to discuss this: https://github.com/LLK/scratch-vm/issues/2137